### PR TITLE
Synchronize changes from alt-1

### PR DESCRIPTION
**Pull request by jovial**
This is an attempt to bring in all changes that have been applied to alt-1 but not yet made it to p3.

### CODE REVIEW

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -196,6 +206,19 @@ kolla_overcloud_inventory_custom_components: "{{ lookup('template', kayobe_confi
...
+      - controllers
+  network:
+    groups:
+      - network
+  compute:
+    groups:
+      - compute
+  monitoring:
+    groups:
+      - controllers
```
Comment (markgoddard) [inline etc/kayobe/kolla.yml]: I think we need to stick with the default here. Monitoring runs on the controller in alt-1, but has its own node in production

File: etc/kayobe/overcloud.yml [hunk 2]
```diff
@@ -24,6 +24,10 @@ overcloud_group_hosts_map:
 #disable_cloud_init:
 disable_cloud_init: True
 
+# To prevent some network issues you can choose to disable cloud-init
+#disable_cloud_init:
+disable_cloud_init: True
```
Comment (markgoddard) [inline etc/kayobe/overcloud.yml]: We've got these lines twice now

File: etc/kayobe/kolla/passwords.yml [hunk 3]
```diff
@@ -1,1018 +1,1018 @@
 $ANSIBLE_VAULT;1.1;AES256
```
Comment (markgoddard) [inline etc/kayobe/kolla/passwords.yml]: Checked the diff and there are no changes in the decrypted file

**Review by markgoddard — CHANGES_REQUESTED** [review]
Nick hasn't deployed the Keystone federation changes in production yet, so we'll need to revert those.

**Comment by jovial** [comment]
Couldn't find a cleaner way than reverting the keystone commit

**Review by markgoddard — APPROVED** [review]

File: etc/kayobe/secrets.yml [hunk 4]
```diff
@@ -1,54 +1,54 @@
 $ANSIBLE_VAULT;1.1;AES256
-64353261323166643863303933386131666130623233303335363830383962383134393537313262
```
Comment (markgoddard) [inline etc/kayobe/secrets.yml:2]: Despite the changes, this is a noop. Not worth worrying about.

File: etc/kayobe/kolla.yml [hunk 5]
```diff
@@ -246,6 +252,7 @@ kolla_enable_grafana: True
 #kolla_enable_influxdb:
 kolla_enable_influxdb: True
 #kolla_enable_ironic:
+kolla_enable_ironic_ipxe: True
```
Comment (markgoddard) [inline etc/kayobe/kolla.yml:255]: We'd need to redeploy ironic to enable ipxe.

**Pull request merged**